package.json

```diff
@@ -21,61 +35,42 @@
     }
   ],
   "files": [
-    "dist/",
-    "src/",
-    "postinstall.js"
+    "src/"
```

Comment:
At that point wouldn't they jus script incle to dist/min so package.json would have no effect.  This prevents people from misusing as an npm module